tools/code-owners-parser/Azure.Sdk.Tools.RetrieveCodeOwners/Program.cs

```diff
@@ -45,30 +43,26 @@ string vsoOwningLabels
             List<string> users = new List<string>();
             List<string> teams = new List<string>();
             List<string> labels = new List<string>();
-
-            if (filteredEntries.Count() > 0)
+            if (filteredEntries != null)
             {
-                Console.WriteLine($"Found a folder to match {target}");
-
-                foreach (var entry in filteredEntries)
+                Console.WriteLine($"Found the closest code owners to match {target}:");
+                Console.WriteLine(String.Join(", ", filteredEntries.Owners));
+                foreach (var alias in filteredEntries.Owners)
                 {
-                    foreach (var alias in entry.Owners)
+                    var userUriStub = $"users/{alias}";
+                    if (VerifyAlias(userUriStub))
```

Comment:
We should probably also add this logic to the code owners library as it seems like something most consumers will want to do.

Reply:
Will put have two helper `ParseAndFindClosestMatch` and `VerifyAlias` to the right class.